app/views/alaveteli_pro/info_request_batches/_embargo_form.html.erb

```diff
@@ -1,7 +1,9 @@
 <% if info_request_batch.embargo_duration.present? %>
 <div class="sidebar__section update-embargo">
   <h2 class="embargo-sidebar-heading">
-    <i class="embargo-indicator embargo-indicator--small"></i>
+    <i class="embargo-indicator embargo-indicator--small"
+       title="<%= I18n.l(info_request_batch.info_requests[0].embargo.publish_at,
```

Comment:
Use `first` instead of `[0]`.

`first` adds a `LIMIT` to the SQL, but the array accessor does not.

Reply:
👍